### PR TITLE
Translate doc url at Crowdin

### PR DESCRIPTION
**Pull request by pierreneter**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
     "del": "^2.2.2",
     "dotenv-safe": "^4.0.4",
     "electron-docs": "^3.0.2",
+    "get-crowdin-file-ids": "github:electron/get-crowdin-file-ids#get-method",
```
Comment (pierreneter) [inline package.json]: Related to: https://github.com/electron/get-crowdin-file-ids/pull/6

File: script/build-module.js [hunk 2]
```diff
@@ -45,18 +50,23 @@ async function parseFile (file) {
...
+    .split('/') // path.sep => /, separator in file.relativePath is just / in any OS
     .slice(2, -1)
-    .join(path.sep)
+    .join('/')
 
   // nice categories for use in nav
   file.categoryFancy = categoryNames[file.category]
 
-  file.href = path.join('/docs', file.category, file.slug)
+  file.href = `/docs/${file.category}/${file.slug}`.replace('//', '/')
```
Comment (zeke) [inline script/build-module.js]: Why is this needed? I think `path.join` will take care of duplicate slashes.

File: script/build-module.js [hunk 3]
```diff
@@ -73,31 +83,49 @@ async function parseFile (file) {
...
     let src = $(el).attr('src')
     const type = hrefType(src)
+
     if (type !== 'relative' && type !== 'rooted') return
 
     // turn `../images/foo/bar.png` into `/docs/images/foo/bar.png`
-    src = path.resolve(dirname, src)
-
-    const newSrc = file.isApiDoc
+    if (src.startsWith('../')) {
```
Comment (zeke) [inline script/build-module.js]: Since this is used in two places, can you generalize it into function like `resolveRelativePath` or similar?

File: stats.json [hunk 4]
```diff
@@ -2,10 +2,10 @@
   {
     "name": "Arabic",
     "code": "ar",
-    "phrases": "8282",
+    "phrases": "8287",
```
Comment (zeke) [inline stats.json]: Can you omit the changes to `stats.json` from this PR? They will happen automatically on build.

File: package.json (hunk 1 above)
Comment (zeke) [reply package.json]: You fix landed in get-crowdin-file-ids@1.1.1. You should now be able to `npm i -D get-crowdin-file-ids@latest` 👍

File: script/build-module.js (hunk 2 above)
Comment (pierreneter) [reply script/build-module.js]: I think `.replace` is simple and just work on both of Windows, Unix, and Linux env.
On Windows, `path.join` will return a path has separators are `\` in it. (must be replaced `\` to `/`, It takes up to 2 steps while we just need to replace it once for simplicity).

File: readme.md [hunk 5]
```diff
@@ -18,11 +18,11 @@ The following languages are currently being translated, but we can
 <!-- start language-table -->
 - [Indonesian](https://crowdin.com/project/electron/id)
 - [Türkçe (Turkish)](https://crowdin.com/project/electron/tr)
-- [中文 (Chinese Simplified)](https://crowdin.com/project/electron/zh-CN)
 - [Español (Spanish)](https://crowdin.com/project/electron/es-ES)
```
Comment (zeke) [inline readme.md]: Can you revert changes to the readme here? These changes will happen in the build.

File: script/build-module.js (hunk 2 above)
Comment (zeke) [reply script/build-module.js]: Ok 👍

File: script/build-module.js [hunk 6]
```diff
@@ -73,31 +83,45 @@ async function parseFile (file) {
...
-    const dirname = path.dirname(file.href)
     let src = $(el).attr('src')
     const type = hrefType(src)
+
     if (type !== 'relative' && type !== 'rooted') return
 
     // turn `../images/foo/bar.png` into `/docs/images/foo/bar.png`
+
     src = path.resolve(dirname, src)
+            // the same with line 88
```
Comment (zeke) [inline script/build-module.js]: All the regexes here make me nervous. At the very least, let's try to minimize them by creating a re-usable function.

Comment (pierreneter) [reply script/build-module.js]: Agree.

File: readme.md (hunk 5 above)
Comment (pierreneter) [reply readme.md]: Sure.

**Comment by pierreneter** [comment]
Ping @zeke.

**Comment by zeke** [comment]
Yo! I'll try to get to this today.
